src/helpers/byText.ts

```diff
@@ -59,7 +62,10 @@ const getNodeByText = (
     }
     return false;
   } catch (error) {
-    throw createLibraryNotSupportedError(error);
+    if (error instanceof Error) {
+      throw createLibraryNotSupportedError(error);
```

Comment:
IMO we should also throw for non-`Error` errors. I'd suggest changing `createLibraryNotSupportedError` to accept `unknown` instead of `Error` and then do some checks inside.

Reply:
This relates to all similar cases in al of the query files.